### PR TITLE
Update most read numerals

### PR DESCRIPTION
**Pull request by thekp**
Resolves n/a

**Overall change:** _Use Latin numerals for Arabic and Urdu_

**Code changes:**

- _remove EasternArabic numerals for Arabic and Urdu._
- _Updating double-digit spacing for Arabic and Urdu._

URDU:
image

ARABIC: 
image

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: packages/components/psammead-most-read/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 2.0.2 | [PR#3175](https://github.com/bbc/psammead/pull/3175) Use Latin numerals for Arabic and Urdu rank |
```
Comment (OlgaLyubin) [inline packages/components/psammead-most-read/CHANGELOG.md]:
```suggestion
| 2.0.2 | [PR#3199](https://github.com/bbc/psammead/pull/3199) Use Latin numerals for Arabic and Urdu rank |
```

**Review by OlgaLyubin — APPROVED** [review]
Looks good on storybook 👍

**Review by DenisHdz — APPROVED** [review]
LGTM

**Comment by thekp** [comment]
The refactor here: https://github.com/bbc/psammead/pull/3201
image

will ensure the numerals are all in line.

**Comment by PriyaKR** [comment]
LGTM on storybook.
Screenshot 2020-03-03 at 16 56 58

**Comment by PriyaKR** [comment]
> The refactor here: #3201
> 
> will ensure the numerals are all in line.

What's the issue?

**Comment by PriyaKR** [comment]
Looks like the padding issue will be fixed forward.
Copying comments from discussion with @thekp 

these changes won’t be pulled in until we bump it in simorgh.so Urdu front page is live now, so i suppose editorial are the people that wanted to change the numerals

**Comment by thekp** [comment]
I've spoken to UX and Product, we are fine with the small spacing issue for now.

**Pull request merged**